app/templates/src/main/java/package/config/_WebMvcConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package <%=packageName%>.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.web.PageableHandlerMethodArgumentResolver;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+import java.util.List;
+
+@Configuration
+public class WebMvcConfiguration extends WebMvcConfigurerAdapter {
+
+    @Bean
+    public PageableHandlerMethodArgumentResolver pageableHandlerMethodArgumentResolver() {
+        PageableHandlerMethodArgumentResolver pageableResolver = new PageableHandlerMethodArgumentResolver();
+        pageableResolver.setPageParameterName("page");
+        pageableResolver.setSizeParameterName("per_page");
+        pageableResolver.setMaxPageSize(100);
```

Comment:
I'm going to remove this to follow our Policy 1 at http://jhipster.github.io/policies.html